activiti-services/activiti-services-audit/activiti-services-audit-jpa/src/main/resources/application.properties

```diff
@@ -22,4 +22,7 @@ keycloakclientpassword=client
 # this user needs to have the realm management roles assigned
 # one option to deal with password being here would be spring-boot-jasypt, keycloak may support others (can it be set as env variable?)
 spring.rabbitmq.host=rabbitmq
-spring.rabbitmq.port=5672
\ No newline at end of file
+spring.rabbitmq.port=5672
```

Comment:
@ryandawsonuk as I mention today, do we really need this? I would try to avoid it.